code/parachain/frame/crowdloan-rewards/src/lib.rs

```diff
@@ -199,6 +203,14 @@ pub mod pallet {
 		/// The unique identifier of this pallet.
 		#[pallet::constant]
 		type PalletId: Get<PalletId>;
+
+		/// The unique identifier for locks maintained by this pallet.
+		#[pallet::constant]
+		type LockId: Get<LockIdentifier>;
+
+		/// If claimed amounts should be locked by the pallet
+		#[pallet::constant]
+		type LockByDefault: Get<bool>;
```

Comment:
Do we really need this flag? Given that there is also `RemoveRewardLocks` which can be set/reset at genesis?

Reply:
I was under the impression that changing the crowdloans genesis config was not an option. Is this not the case?
@vimukthi-git

Reply:
Yeah, I remember now, keep forgetting that detail :sweat_smile: